_posts/0000-01-02-nocodemike.md

```diff
@@ -0,0 +1,8 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
#This is what needs to happen. This is a beautiful table.

#2 My first contribution

![28c0f313-c6b3-4a5b-a1d8-561d2e723d21](https://user-images.githubusercontent.com/7183947/115163601-ca84c180-a05e-11eb-9843-bdb2627b186b.png)